schema-loader/src/main/java/com/scalar/db/schemaloader/command/SpecificStorageCommandBase.java

```diff
@@ -0,0 +1,30 @@
+package com.scalar.db.schemaloader.command;
+
+import com.scalar.db.schemaloader.SchemaLoaderException;
+import com.scalar.db.schemaloader.core.SchemaOperator;
+import com.scalar.db.schemaloader.core.SchemaOperatorException;
+import com.scalar.db.schemaloader.core.SchemaOperatorFactory;
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.Properties;
+
+public abstract class SpecificStorageCommandBase {
```

Comment:
If we have this base class, I think we can move the `deleteTables` and `schemaFile` fields to this class. After that, we can also remove the `schemaFile` and `deleteTables` arguments from the `execute()` method. What do you think?

Reply:
Thanks for suggestion, I moved them to the base class [commit](https://github.com/scalar-labs/scalardb/pull/366/commits/219473b5710f1f7154ae5075eaa1531d48b854d6)